src/export.js

```diff
@@ -1,24 +1,31 @@
-// Deprecated
-// Extend assert methods to QUnit for Backwards compatibility
-(function() {
-	var i,
-		assertions = Assert.prototype;
-
-	function applyCurrent( current ) {
-		return function() {
-			var assert = new Assert( QUnit.config.current );
-			current.apply( assert, arguments );
-		};
-	}
-
-	for ( i in assertions ) {
-		QUnit[ i ] = applyCurrent( assertions[ i ] );
-	}
-})();
+function globalWarning( key ) {
+	return function() {
+		QUnit.warning( "exported-globals" );
```

Comment:
I'd prefer distinct warnings for every method, and for calls on `QUnit` vs. the global object (`qunit-ok`, `global-ok`, `qunit-notOk`, `global-notOk`, etc.).